src/components/SettingsPanel/index.jsx

```diff
@@ -0,0 +1,66 @@
+import cls from 'classnames';
+import { useState, useEffect } from 'react';
+import store from '../../store';
+import Input from '../Input';
+import Button from '../Button';
+import { flex, layout, margin, padding, colors, typography as type} from '../../theme';
+import styles from './styles.module.css';
+import ProjectsFolder from './ProjectsFolder';
+
+export default function SettingsPanel() {
```

Comment:
Is it supposed to look like this? Feels like some of the spacing is a little off between the inputs. Also don't think we need those dividers, the inputs themselves serve as dividing elements. 


<img width="775" alt="image" src="https://user-images.githubusercontent.com/6759188/204638795-21f27f51-bb9b-4a77-b3c4-8fa79ccef100.png">